transports/jms/src/test/java/org/mule/compatibility/transport/jms/integration/JmsRequestResponseTestCase.java

```diff
@@ -25,7 +25,7 @@ protected String getConfigFile() {
   @Test
   public void testNotSendiningCorrelationIDWithTemporaryQueue() throws Exception {
     InternalMessage response =
-        flowRunner("JMSNoCorrelationIDTemporaryQueue").nonBlocking().withPayload(TEST_MESSAGE).run().getMessage();
+        flowRunner("JMSNoCorrelationIDTemporaryQueue").withPayload(TEST_MESSAGE).run().getMessage();
```

Comment:
should this class extend the new abstract class to continue testing the non blocking scenarios?